physics/escape_velocity.py

```diff
@@ -0,0 +1,74 @@
+"""
+Title: Calculate Escape Velocity
+
+Description:
+    This algorithm calculates the escape velocity required for an object to
+    leave the gravitational influence of a celestial body.
+"""
+
+import math
+
+
+def calculate_escape_velocity(
+    mass_celestial_body: float, distance_from_center: float
+) -> float:
+    """
+    Calculate the escape velocity required for an object to leave the
+    gravitational influence of a celestial body.
+
+    Formula:
+    escape_velocity = sqrt(2 * gravitational_constant * mass_celestial_body /
+    distance_from_center)
+
+    Parameters:
+    - mass_celestial_body (float): Mass of the celestial body in kilograms.
+    - distance_from_center (float): Distance from the center of the celestial
+    body in meters.
+
+    Constants:
+    - gravitational_constant (float): Universal gravitational constant in SI
+    units (6.67430e-11 m³/kg/s²).
+
+    Examples:
+    >>> calculate_escape_velocity(mass_celestial_body=5.972e24,
+    distance_from_center=6.371e6)
```

Comment:
```suggestion
    >>> calculate_escape_velocity(mass_celestial_body=5.972e24,
    ... distance_from_center=6.371e6)
```
You need to use `...` in multi-line doctests to avoid syntax errors